regression/regression_param.sh

```diff
@@ -84,11 +84,11 @@ case $regtest in
 
     ;;
 
-    rrfs_3denvar_glbens)
+    rrfs_3denvar_rdasens)
 
         if [[ "$machine" = "Hera" ]]; then
-           topts[1]="0:15:00" ; popts[1]="5/4/"  ; ropts[1]="/1"
-           topts[2]="0:15:00" ; popts[2]="10/4/"  ; ropts[2]="/1"
+           topts[1]="0:05:00" ; popts[1]="40/3/"  ; ropts[1]="/1"
```

Comment:
sorry, i forgot what  "40/3" means.    40 nodes with 3 tasks per node?  If so, requesting 40 nodes is  too much a for ctest?

Reply:
it is 40 cores per node (ppn) and three nodes. Hera has 40 cores per node.